tests/test_alternative_citations.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+from nose.tools import *  # PEP8 asserts
+
+from website.project.model import AlternativeCitation
+
+from modularodm.exceptions import ValidationError
+
+from tests.base import OsfTestCase
+
+from tests.factories import UserFactory, ProjectFactory
+
+class ModelTests(OsfTestCase):
+
+    def setUp(self):
+        OsfTestCase.setUp(self)
+        self.user = UserFactory()
+        self.node = ProjectFactory(creator=self.user)
+        self.citation = AlternativeCitation(name='Initial Citation', text='This is my test citation')
+        self.citation.save()
+        self.node.alternativeCitations.append(self.citation)
+
+    def test_model_success(self):
+        alt_citation = AlternativeCitation(name='test', text='citation')
+        alt_citation.save()
+        self.node.alternativeCitations.append(alt_citation)
+        assert_equal(len(self.node.alternativeCitations), 2)
+
+    def test_model_no_name(self):
+        alt_citation = AlternativeCitation(text='citation')
+        try:
+            alt_citation.save()
+            self.node.alternativeCitations.append(alt_citation)
+        except ValidationError:
```

Comment:
For the tests are supposed to raise a `ValidationError` they should assert that the error gets raised instead of catching the exception.